everyvoice/cli.py

```diff
@@ -268,17 +271,50 @@ class ModelTypes(str, Enum):
     short_help="Export your EveryVoice models",
 )
 
-app.command(
-    short_help="Segment a long audio file",
-    name="segment",
+# Add the segment commands
+segment_group = typer.Typer(
+    pretty_exceptions_show_locals=False,
+    context_settings={"help_option_names": ["-h", "--help"]},
+    rich_markup_mode="markdown",
+    cls=TyperGroupOrderAsDeclared,
+    help="""
+    # Segment Help
+
+        - **align** --- This command will align a long audio file with some text.
+
+        - **extract** --- This command will take the alignment from the `align` command and extract it into multiple utterances in the format required for training a TTS system
+    """,
+)
+
+
+segment_group.command(
+    short_help="Align a long audio file with some text",
+    name="align",
     help="""
     # Segmentation help
 
-    This command will segment a long audio file into multiple utterances which is required for training a TTS system.
+    This command will align a long audio file with some text.
```

Comment:
Here you should add a somewhat longer description of the granularity of segmentation one can expect.